--- a/src/bin/exactness/recorder.c.orig	2026-06-17 17:06:21.157823964 +0300
+++ b/src/bin/exactness/recorder.c	2026-06-17 17:08:00.180831942 +0300
@@ -301,7 +301,7 @@
 
 #ifdef HAVE_DLSYM
 # define ORIGINAL_CALL_T(t, name, ...) \
-   t (*_original_init_cb)(...); \
+   t (*_original_init_cb)(); \
    _original_init_cb = dlsym(RTLD_NEXT, name); \
    original_return = _original_init_cb(__VA_ARGS__);
 #else
--- a/src/bin/exactness/player.c.orig	2026-06-17 17:09:44.231840326 +0300
+++ b/src/bin/exactness/player.c	2026-06-17 17:07:53.704831420 +0300
@@ -1040,7 +1040,7 @@
 }
 #ifdef HAVE_DLSYM
 # define ORIGINAL_CALL_T(t, name, ...) \
-   t (*_original_init_cb)(...); \
+   t (*_original_init_cb)(); \
    _original_init_cb = dlsym(RTLD_NEXT, name); \
    original_return = _original_init_cb(__VA_ARGS__);
 #else
